src/tools/generate/generate_sitemap.js

```diff
@@ -6,7 +6,6 @@ const { get_static_lang_files } = require('./shared');
 const min_publish_date = '2019-11-11';
 const sitemap_template = `templates/sitemap.ejs.xml`;
 const sitemap_path = `templates/sitemap.xml`;
-const ebook_path = "static/pdfs/web_almanac_";
```

Comment:
Should we keep the ebooks in the sitemap, but use their absolute URLs on the CDN? Not sure if that's a thing.

Reply:
AFAIK that’s not a thing. However Google should still crawl them anyway. Just might take a little longer to recrawl without a site map to give a clear indication they’ve been updated.